requirements/constraints.txt

```diff
@@ -43,32 +43,15 @@ pytest-asyncio<0.24
 #
 # Bugs
 #
-
+httpx<0.28 # Waiting for fix in respx: https://github.com/lundberg/respx/pull/278
```

Comment:
I would suggest to do `httpx!=0.2.8` to not forget to update it later on. Actually there is already a PR on the respx side for this, so I am guessing `httpx==0.2.9` will work once that comes out.

Reply:
ok. but it is 0.28 not 0.2.8 :-) you got me confused